catkit/hardware/newport/NewportPicomotorController.py

```diff
@@ -195,7 +195,9 @@ def command(self, cmd_key, axis, value):
         set_value = float(self._send_message(get_message, 'get')) - float(initial_value)
         
         if float(set_value) != value:
-            self.log.error(f'Something is wrong, {set_value} != {value}') 
```

Comment:
I encountered this error a handful of times when testing https://github.com/spacetelescope/hicat-package/pull/310 during the power and auto compensation phase of the npoint controller.

Doubling ``sleep_per_step`` from ``0.0005`` to ``0.001`` seemed to fix this. I'm not sure though if we'll need to make this a permanent change or whether it's indicative of dying hardware 😬 .

Reply:
@RemiSoummer Have you ever seen this before?